recipes-core/system-api/system-api_v0.7.0.bb

```diff
@@ -6,12 +6,13 @@ LIC_FILES_CHKSUM = "file://src/${GO_WORKDIR}/LICENSE;md5=c7bc88e866836b5160340e6
 inherit go-mod update-rc.d
 
 INITSCRIPT_NAME = "system-api-init"
-INITSCRIPT_PARAMS = "defaults 81"
+INITSCRIPT_PARAMS = "defaults 88"
```

Comment:
shouldn't the encrypted disk be before the system-api? 
So disk-encryption init script should be 88 and system api should on the run level 89